android/gradle.properties

```diff
@@ -13,6 +13,7 @@ org.gradle.parallel=true
 # Optimized for GitHub runners (4 cores, 16GB RAM)
 org.gradle.jvmargs=-Xmx8g -XX:MaxMetaspaceSize=1g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.caching=true
+# org.gradle.configuration-cache=true # TODO: configuration cache still not ready for prime time
```

Comment:
let's try clearing the other kind of cache on the workspace and see if this one is any faster